Week2/four.js

```diff
@@ -0,0 +1,21 @@
+// 4
+let y = "Hey Guys";
+
+// 4.1
+
+console.log("the value of the string will be: undefined ");
```

Comment:
Are you sure that this is what the value will be?

Reply:
if i write ; console.log (y) and run the program , then the value will be hey guys

Am I right?

Reply:
You are right, so you can update your code (or change the "hey guys")